normalizer_lib.py

```diff
@@ -83,24 +83,24 @@ def verbalizable(self):
     # e.g. "hi      there" -> "hi there"
 
 
-    def remove_extra_whitespace(self, fst: Fst) -> Fst:
+    def remove_extra_whitespace(self) -> Fst:
         "Removes extra whitespace."
         remove_extra_whitespace = cdrewrite(
             pynutil.delete(byte.SPACE),
             "",
             byte.SPACE,
             self.SIGMA_STAR)
-        return (fst @ remove_extra_whitespace).optimize()
+        return remove_extra_whitespace
 
 
     # Language-specific formatting fixes
 
 
-    def language_specific_fixes(self, fst: Fst) -> Fst:
+    def language_specific_fixes(self) -> Fst:
         "Applies language-specific formatting fixes."
         if self.language_specific_preprocessing:
-            return (fst @ self.language_specific_preprocessing).optimize()
-        return fst
+            return self.language_specific_preprocessing
+        return self
```

Comment:
I don't think you need return self, I think you can just return self.language_specific_preprocessing.

Reply:
Fixed by using `DUMMY_REWRITE`!